src/libraries/System.IO.Pipes.AccessControl/src/System.IO.Pipes.AccessControl.csproj

```diff
@@ -1,20 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>System.IO.Pipes.AccessControl</AssemblyName>
-    <IncludeDefaultReferences>false</IncludeDefaultReferences>
     <IsPartialFacadeAssembly Condition="'$(TargetsWindows)' == 'true'">true</IsPartialFacadeAssembly>
     <OmitResources Condition="'$(TargetsWindows)' == 'true'">true</OmitResources>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsWindows)' != 'true'">SR.PlatformNotSupported_AccessControl</GeneratePlatformNotSupportedAssemblyMessage>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;netcoreapp2.1;$(NetCoreAppCurrent)</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+  <PropertyGroup>
+    <IncludeDefaultReferences Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">false</IncludeDefaultReferences>
+    <AssemblyVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">4.0.3.0</AssemblyVersion>
```

Comment:
@ericstj believes that we should be able to get away without setting this and using 5.0.0.0

Reply:
I don't think so, because we are still harvesting `runtimes/win/lib/netcoreapp2.1` asset that is 4.0.3.0, so if we make `lib/netcoreapp2.1` be 5.0.0.0 our packaging validation will fail saying that the compile time asset was found to support 5.0.0.0 but the runtime asset was found to support 4.0.3.0. If we wanted to do this, we would have to go and bring back the netcoreapp2.1-Windows_NT live build.

Reply:
Ah that's right.

Reply:
I thought we only needed to build `net5.0` assets since that was the only place we could add surface area. @carlossanlop?

Reply:
This is needed in order to avoid bringing the refs. This asset will only be used for compile time; but I guess that's a good point.

We would need to exclude this APIs for older targets so that they're not available there.

Reply:
See here: https://github.com/dotnet/runtime/pull/38177#discussion_r443864354

Reply:
Actually, maybe the goal of avoiding using refs here might have been bad advice on my part.  This is a full-facade in src, but contains typedefs in ref.  By only shipping the src assembly we're shipping dangling type-forwards to the compiler 🤦‍♂️

Reply:
Ah that's right... dang it. No worries, none of use caught it either 😄